README.md

```diff
@@ -45,60 +81,25 @@ In case your browser doesn't have support for [Fetch](https://developer.mozilla.
-
-    > Learn how to [create an instance of ImplicitMSALAuthenticationProvider](./docs/ImplicitMSALAuthenticationProvider.md).
-
-User can integrate own preferred authentication library by implementing `IAuthenticationProvider` interface. Refer implementing [Custom Authentication Provider](./docs/CustomAuthenticationProvider.md).
-
-### 3. Initialize a Microsoft Graph Client object with an authentication provider
-
-An instance of the **Client** class handles requests to Microsoft Graph API and processing the responses. To create a new instance of this class, you need to provide an instance of [`IAuthenticationProvider`](src/IAuthenticationProvider.ts) which needs to be passed as a value for `authProvider` key in [`ClientOptions`](src/IClientOptions.ts) to a static initializer method `Client.initWithMiddleware`.
+To call Microsoft Graph, your app must acquire an access token from the Microsoft identity platform.
+Learn more about this -
+- [Authentication and authorization basics for Microsoft Graph](https://docs.microsoft.com/en-us/graph/auth/auth-concepts)
+- [Register your app with the Microsoft identity platform](https://docs.microsoft.com/en-us/graph/auth/auth-concepts)
 
-#### For browser environment
+### 2. Create a Client Instance
 
-```typescript
-const options = {
-	authProvider, // An instance created from previous step
-};
-const Client = MicrosoftGraph.Client;
-const client = Client.initWithMiddleware(options);
-```
-
-#### For node environment
-
-```typescript
-import { Client } from "@microsoft/microsoft-graph-client";
+The Microsoft Graph client is designed to make it simple to make calls to Microsoft Graph. You can use a single client instance for the lifetime of the application.
 
-const options = {
-	authProvider, // An instance created from previous step
-};
-const client = Client.initWithMiddleware(options);
-```
+For information on how to create a client instance, see [Creating Client Instance](./docs/CreatingClientInstance.md)
 
-For more information on initializing client, refer [this document](./docs/CreatingClientInstance.md).
 
-### 4. Make requests to the graph
+### 3. Make requests to the graph
 
-Once you have authentication setup and an instance of Client, you can begin to make calls to the service. All requests should be start with `client.api(path)` and end with an [action](./docs/Actions.md).
+Once you have authentication setup and an instance of Client, you can begin to make calls to the service. All requests should start with `client.api(path)` and end with an [action](./docs/Actions.md).
 
-Getting user details
+Getting user details -
```

Comment:
Q: what is the trailing ` -` sign for?

Reply:
Indicating the the following sample is for Getting user details